packages/scripts/entrypoint.sh

```
@@ -1,20 +1,24 @@
-log() {
-    echo "[$(date +'%Y-%m-%d %H:%M:%S')] $1"
-}
+ENTCMD_PREFIX=""
 
 # Ensures user directory ownership
 chown_user_directory() {
-    local user_group="${USER}:${GID}"
-    if ! chown -h --no-preserve-root "$user_group" "${HOME}" 2>/dev/null; then
-        echo "Error: Failed to change ownership of ${HOME} to ${user_group}" >&2
+    if ! $ENTCMD_PREFIX chown "${NESTRI_USER}:${NESTRI_USER}" "${NESTRI_HOME}" 2>/dev/null; then
+        echo "Error: Failed to change ownership of ${NESTRI_HOME} to ${user_group}" >&2
```

Shellcheck (0.10.0)

[warning] user_group is referenced but not assigned.

(SC2154)